modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -36,18 +36,33 @@ static inline float tsdfToFloat(TsdfType num)
 {
     return float(num) * (-1.f / 128.f);
 }
-
+*/
```

Comment:
Extra comments should be removed, here and everywhere

Reply:
done, but not all